public/css/icinga/flyover.less

```diff
@@ -0,0 +1,16 @@
+/*! Icinga Web 2 | (c) 2017 Icinga Development Team | GPLv2+ */
+
+.flyover:not(.flyover-expanded) .flyover-content {
+  display: none;
+}
+
+.flyover {
+  position: relative;
+
+  .flyover-content {
+    position: absolute;
```

Comment:
Please move `position: absolute` to the bottom of this CSS block. I like that most of our CSS is ordered alphabetically or by its function.